docker/docker-compose.yml

```diff
@@ -89,87 +103,95 @@ services:
+  #     "--apiPassword=apitest",
+  #     "--apiPort=9998",
+  #     "--walletRpcBindPort=38090",
+  #     "--passwordRequired=false",
+  #   ]
+  #   depends_on:
+  #     - seed1
+  #   # ports:
+  #   #   - "127.0.0.1:38090:38090"
 
-  user1:
-    build: haveno/
-    container_name: user1
-    entrypoint: [ "./haveno-daemon" ]
-    command: [
-      "--baseCurrencyNetwork=XMR_LOCAL",
-      "--useLocalhostForP2P=true",
-      "--useDevPrivilegeKeys=true",
-      "--nodePort=5555",
-      "--appName=haveno_user1",
-      "--apiPassword=apitest",
-      "--apiPort=9999",
-      "--walletRpcBindPort=38091",
-      "--passwordRequired=false",
-    ]
-    network_mode: "host"
-    depends_on:
-      - seed1
+  # user1:
+  #   build: haveno/
+  #   container_name: user1
+  #   entrypoint: [ "./haveno-daemon" ]
+  #   command: [
+  #     "--baseCurrencyNetwork=XMR_LOCAL",
+  #     "--useLocalhostForP2P=true",
+  #     "--useDevPrivilegeKeys=true",
+  #     "--nodePort=5555",
+  #     "--appName=haveno_user1",
+  #     "--apiPassword=apitest",
+  #     "--apiPort=9999",
+  #     "--walletRpcBindPort=38091",
+  #     "--passwordRequired=false",
+  #   ]
+  #   depends_on:
+  #     - seed1
+  #   # ports:
+  #   #   - "127.0.0.1:38091:38091"
 
-  user2:
-    build: haveno/
-    container_name: user2
-    entrypoint: [ "./haveno-daemon" ]
-    command: [
-      "--baseCurrencyNetwork=XMR_LOCAL",
-      "--useLocalhostForP2P=true",
-      "--useDevPrivilegeKeys=true",
-      "--nodePort=6666",
-      "--appName=haveno_user2",
-      "--apiPassword=apitest",
-      "--apiPort=10000",
-      "--walletRpcBindPort=38092",
-      "--passwordRequired=false",
-    ]
-    network_mode: "host"
-    depends_on:
-      - seed1
+  # user2:
+  #   build: haveno/
+  #   container_name: user2
+  #   entrypoint: [ "./haveno-daemon" ]
+  #   command: [
+  #     "--baseCurrencyNetwork=XMR_LOCAL",
+  #     "--useLocalhostForP2P=true",
+  #     "--useDevPrivilegeKeys=true",
+  #     "--nodePort=6666",
+  #     "--appName=haveno_user2",
+  #     "--apiPassword=apitest",
+  #     "--apiPort=10000",
+  #     "--walletRpcBindPort=38092",
+  #     "--passwordRequired=false",
+  #   ]
+  #   depends_on:
+  #     - seed1
+  #   # ports:
+  #   #   - "127.0.0.1:38092:38092"
 
   pricenode:
     build: pricenode/
     container_name: pricenode
     entrypoint: [ "./haveno-pricenode" ]
-    network_mode: "host"
 
   funding:
     build: haveno/
     container_name: funding_wallet
     entrypoint: [ "./.localnet/monero-wallet-rpc" ]
     command: [
       "--testnet",
-      "--daemon-address=http://localhost:28081",
+      "--daemon-address=http://node2:28081",
+      "--confirm-external-bind",
+      "--rpc-bind-ip=0.0.0.0",
       "--rpc-bind-port=28084",
       "--rpc-login=rpc_user:abc123",
-      "--rpc-access-control-origins=http://localhost:8080",
-      "--wallet-dir=./.localnet",
+      "--rpc-access-control-origins='*'",
+      "--wallet-dir=./.localnet/funding_wallet",
```

Comment:
Why move the funding wallet to this folder?

Reply:
This is for docker to place the artifacts into proper local persistent directory